src/App.jsx

```diff
@@ -14,57 +16,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(goods[8]);
```

Comment:
Let's search for our item by name.
If 'Jam' moves to another place - ur logic will break
Change ur logic
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```